Meeting notes for the Steering Council for August 2024 (#257)
python/steering-council: updates/2024-08-steering-council-update.md

# Steering Council Updates for August 2024

## 2024-08-07

- Conclusion of the Code of Conduct violation
- Discussion on [PEP 667 - Consistent views of namespaces](https://peps.python.org/pep-0667/) semantic changes

## 2024-08-14

> - Deliberation of the decision based on the Code of Conduct violation

Reviewer: ```suggestion
- Deliberation of the decision regarding the Code of Conduct violation
```

- Discussion on how to make the Steering Council more effective by suggesting a different body parallel with the Steering Council to deal with social matters

> - Highlight on the need to make clarification on the following different bodies Code of Conduct Working Group, Steering Council and Python Software Foundation.

Reviewer: ```suggestion
- Highlight on the need to make clarification on the following different bodies: Code of Conduct Working Group, Steering Council and Python Software Foundation
```

- Discussion on the resignation of a Core Python Developer

> - Mentoring survey: still to be released.

Reviewer: nit: the other bullets in this list don't end in periods

```suggestion
- Mentoring survey: still to be released
```

- Discussion on [PEP 741 - Python Configuration C API](https://peps.python.org/pep-0741/)

## 2024-08-21

- Discussion on [PEP 741 - Python Configuration C API](https://peps.python.org/pep-0741/)
- Discussion on having overlapping Steering Council terms for continuity’s sake

> - Discussion on mentorship, its role and how the Steering Council can improve mentorship for some at risk packages/libraries

Reviewer: ```suggestion
- Discussion on mentorship, its role and how the Steering Council can improve mentorship for some at-risk packages/libraries
```

## 2024-08-28

- Cleared out pending tasks and re-evaluated the priority tasks
- Accepted [PEP 741 - Python Configuration C API](https://peps.python.org/pep-0741/)

> - Tabled discussion of [PEP 12 – Sample reStructuredText PEP Template](https://peps.python.org/pep-0012/) changes (regrouping rejection rationale and publishing new modules to PyPI) for the Core Dev sprint

Reviewer: ```suggestion
- Tabled discussion of [PEP 12 – Sample reStructuredText PEP Template](https://peps.python.org/pep-0012/) changes (regrouping rejection rationale, and adding a discussion of publishing new modules to PyPI) for the Core Dev sprint
```

> - Discussion on publishing to PyPI, specifically getting approval and process or flow of release

Reviewer: ```suggestion
- Discussion on publishing to PyPI, specifically:
  - Getting approval
  - Process or flow of release
```

- Re-review of [PEP 667 - Consistent views of namespaces](https://peps.python.org/pep-0667/)
- Discussion on more explicit authority for the Release Manager’s role going forward
- Highlight on the ongoing [PEP 13 – Python Language Governance](https://peps.python.org/pep-0013/) discussion on potential changes

> - PEP for PyREPL

Reviewer: This is a bit vague... I assume this is what happened? 😄

```suggestion
- Discussion on whether there should be a PEP for PyREPL
```

## 2024-08-21

- Discussion on [PEP 741 - Python Configuration C API](https://peps.python.org/pep-0741/)
- Discussion on having overlapping Steering Council terms for continuity’s sake
- Discussion on mentorship, its role and how the Steering Council can improve mentorship for some at-risk packages/libraries

## 2024-08-28

- Cleared out pending tasks and re-evaluated the priority tasks
- Accepted [PEP 741 - Python Configuration C API](https://peps.python.org/pep-0741/)
- Tabled discussion of [PEP 12 – Sample reStructuredText PEP Template](https://peps.python.org/pep-0012/) changes (regrouping rejection rationale, and adding a discussion of  publishing new modules to PyPI) for the Core Dev sprint
- Discussion on publishing to PyPI, specifically:
  - Getting approval
  - Process or flow of release
- Re-review of [PEP 667 - Consistent views of namespaces](https://peps.python.org/pep-0667/)
- Discussion on more explicit authority for the Release Manager’s role going forward
- Highlight on the ongoing [PEP 13 – Python Language Governance](https://peps.python.org/pep-0013/) discussion on potential changes
- Discussion on whether there should be a PEP for PyREPL

> - Discussion on how to know when something needs a PEP or not and update [PEP 2 –Procedure for Adding New Modules](https://peps.python.org/pep-0002/)

Reviewer 2: ```suggestion
- Discussion on how to know when something needs a PEP or not and update [PEP 2 – Procedure for Adding New Modules](https://peps.python.org/pep-0002/)
```
